client/services/utils/localSettings.js

```diff
@@ -0,0 +1,143 @@
+const STORAGE_KEY = 'localsettings'
+const DATA_TYPE_KEY = 'dataType'
+const VALUE_KEY = 'value'
+class LocalSettings {
```

Comment:
Instead of using self-made listeners that we'll need to maintain, use the `EventEmitter` class, like used in [Refidentity](https://github.com/FirstLegoLeague/scoring/blob/master/client/services/scoresheet/ref_identity.js).